### DISCUSSION TITLE
Servo remapping

### DISCUSSION THREAD

**Discussion by TwoToneEddy** [iNavFlight/inav#7861] [Q&A]
Hi,
Has resource remapping been removed? I swear I used to be able to do "resource SERVO3 OUT none" to free and then map it to something else.
Thanks
Lee

**Comment by MrD-RC** [comment]
You could never remap in iNav.

**Reply by TwoToneEddy** [reply]
Ah ok I'm getting confused with betaflight. It looks like th configurator is remapping though when you choose the model type. When I select wing it only maps 2 servos, pins A08 and A09 in my case with A10 as free. When I change the model to a traditional plane I see another servo on A10. It must be possible.
